### PR TITLE
Fix examples structure of TS

### PR DESCRIPTION
**Pull request by hokeun**
Fix https://github.com/lf-lang/playground-lingua-franca/issues/46

### CODE REVIEW

**Comment by lhstrh** [comment]
These renames are not fully correct. Each project should have an src directory with the LF sources in it.

**Comment by hokeun** [comment]
> These renames are not fully correct. Each project should have an src directory with the LF sources in it.

Let me know if it's correct now!

File: examples/TypeScript/HTTPServerReactor/src/gcp/Dockerfile
Comment (lhstrh) [inline examples/TypeScript/HTTPServerReactor/src/gcp/Dockerfile:1]: This should be in `HTTPServerReactor/gcp/`.

File: examples/TypeScript/HTTPServerReactor/src/gcp/README.md
Comment (lhstrh) [inline examples/TypeScript/HTTPServerReactor/src/gcp/README.md:1]: This should be in `HTTPServerReactor/gcp/`.

**Review by lhstrh — CHANGES_REQUESTED** [review]
Looks good except for two files that still need to be moved.

File: examples/TypeScript/HTTPServerReactor/src/gcp/Dockerfile
Comment (hokeun) [reply examples/TypeScript/HTTPServerReactor/src/gcp/Dockerfile:1]: Thanks, done.

File: examples/TypeScript/HTTPServerReactor/src/gcp/README.md
Comment (hokeun) [reply examples/TypeScript/HTTPServerReactor/src/gcp/README.md:1]: Thanks, done.

**Review by lhstrh — APPROVED** [review]
Perfect. Thanks, @hokeun!

**Pull request merged**